cmd/k8s-deployment-scaler/main.go

```diff
@@ -245,8 +241,7 @@ func handleGetReplicaCount(w http.ResponseWriter, r *http.Request) {
 		return
```

Comment:
Depending on the POST/GET request with missing query params error is different.
Validation logic is the same, it's a good candidate to move into separate function.


```
➜  k8s-deployment-scaler git:(development/helm) curl -X GET "https://localhost:8443/replica-count" --cert ./certs/client-cert.pem --key ./certs/client-key.pem --cacert ./certs/ca-cert.pem                             
{"message":"Both namespace and deployment must be specified","code":400}
```

```
➜  k8s-deployment-scaler git:(development/helm) curl -X POST "https://localhost:8443/replica-count" --cert ./certs/client-cert.pem --key ./certs/client-key.pem --cacert ./certs/ca-cert.pem                           
{"message":"Missing query parameters","code":400}
```

Reply:
I agree, I will create a `validateQueryParams` function.